src/chart-review/chart-review.component.tsx

```diff
@@ -42,7 +41,7 @@ export default function ChartReview(props: any) {
   }
 
   React.useEffect(() => {
-    const views: ViewType[] = config.primaryNavbar.map(item => {
+    const views: View[] = config.primaryNavbar.map(item => {
       let view = getView(item.view, config, defaultPath + item.path);
       if (view && view.component) item.component = view.component;
```

Comment:
nit: I prefer having if statements with curly braces. No matter how small.

Reply:
I can try to change. It may be that prettier is doing this automatically.